docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to provision a cluster, and to deploy and update Kyma with alpha commands.
+
+## Provision a cluster
+To provision a kubernetes cluster, run:
```

Comment:
```suggestion
To provision a Kubernetes cluster, run:
```

Reply:
Arrgh I thought I had fixed this. Thanks for noticing again.